test/test_shownoter.py

```diff
@@ -232,7 +232,7 @@ def test_format_links_with_default_title_if_title_not_found(monkeypatch):
     text = "link.com"
     results = shownoter.format_links_as_hash(text)
     assert 1 == len(results)
-    assert "link.com" == results[0]["title"]
+    assert "http://link.com" == results[0]["title"]
```

Comment:
This is related to your other question. because of where it is catching the blank title, it is getting the transformed url rather than the original string.